mk/artifact.mk

```diff
@@ -0,0 +1,57 @@
+USE_PREBUILT ?= 1
+
+CC ?= gcc
+CROSS_COMPILE ?= riscv-none-elf-
+
+BINDIR := $(abspath $(OUT))
+
+TEST_SUITES += \
+    ansibench \
+	rv8-bench
+
+# "ieee754" needs F extension
+# "smolnes", "ticks" have inline assembly and only work in riscv
+TESTBENCHES += \
+    captcha \
+    donut \
+    fcalc \
+    hamilton \
+    $(ieee754) \
+    jit \
+    lena \
+    line \
+    maj2random \
+    mandelbrot \
+    nqueens \
+    nyancat \
+    pi \
+    puzzle \
+    qrcode \
+    richards \
+    rvsim \
+    $(smolnes) \
+    spirograph \
+    $(ticks) \
+    uaes
+
+ifeq ($(USE_PREBUILT),1)
```

Comment:
Check `mk/common.mk` and use `$(call has, ...)`.

Reply:
The `has` macro is to determine the enabled option of rv32emu such as `ENABLE_GDB`, `ENABLE_JIT`, etc,

Reply:
> The `has` macro is to determine the enabled option of rv32emu such as `ENABLE_GDB`, `ENABLE_JIT`, etc,

No, check the following:
```
POSITIVE_WORDS = 1 true yes                                                                                                                                   
define has
$(if $(filter $(firstword $(ENABLE_$(strip $1))), $(POSITIVE_WORDS)),1,0)
endef
```

Using `ENABLE_PREBUILT` consistently will simplify reusing this macro.